web/src/lib/utils/thumbnail-util.spec.ts

```diff
@@ -40,25 +40,16 @@ describe('getAltText', () => {
     'generates correctly formatted alt text when isVideo=$isVideo, city=$city, country=$country, people=$people.length',
     ({ isVideo, city, country, people, expected }) => {
       const asset = {
-        exifInfo: { city, country },
+        text: { city, country, people: people?.map((person: { name: string }) => person.name) ?? [] },
```

Comment:
Why can the type of `person` not be inferred from the `people` array here? I.e. why do you need the extra type hint `person: { name: string }`?